content/base-l10n/docs/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+untranslated: 1
+title: Documentation
+top_graphic: 1
+menu: "main"
+menu:
+  main:
+    weight: 10
+lastmod: 2019-05-15
+---
+
+# Overview
+
+* [Getting Started]({{< ref "/getting-started.md" >}})
+* [How Let's Encrypt Works]({{< ref "/how-it-works.md" >}})
+* [Frequently Asked Questions (FAQ)]({{< ref "/docs/faq.md" >}})
+* [Glossary]({{< ref "/docs/glossary.md" >}})
+
+# Subscriber Information
+
+* [ACME Client Implementations]({{< ref "/docs/client-options.md" >}})
+* [Rate Limits]({{< ref "/docs/rate-limits.md" >}})
+* [Expiration Emails]({{< ref "/docs/expiration-emails.md" >}})
+
+# Advanced Subscriber Information
+
+* [Staging Environment]({{< ref "/docs/staging-environment.md" >}})
+* [Certificate Compatibility]({{< ref "/docs/cert-compat.md" >}})
+* [Chain of Trust (Root and Intermediate Certificates)]({{< ref "/certificates.md" >}})
+* [Upcoming Features]({{< ref "/upcoming-features.md" >}})
+* [Revoking Certificates]({{< ref "/docs/revoking.md" >}})
+* [Certificate Authority Authorization]({{< ref "/docs/caa.md" >}})
+* [Certificates for localhost]({{< ref "/docs/certificates-for-localhost.md" >}})
+* [Best Practice - Keep Port 80 Open]({{< ref "/docs/allow-port-80.md" >}})
+* [Challenge Types]({{< ref "/docs/challenge-types.md" >}})
+* [Certificate Transparency (CT) Logs]({{< ref "/docs/ct-logs.html" >}})
+
+# Client Developer Information
+
+* [Client and Large Provider Integration Guide]({{< ref "/docs/integration-guide.md" >}})
+* [ACME Protocol Updates]({{< ref "/docs/acme-protocol-updates.md" >}})
+* [Differences from Current ACME Draft](https://github.com/letsencrypt/boulder/blob/master/docs/acme-divergences.md)
+* [Finding Account IDs]({{< ref "/docs/account-id.md" >}})
```

Comment:
Do you really want to copy all this actual text? Seems like it would quickly get out of sync.

Reply:
For now, yes: when a new docs page is created, the link must be added in the `/docs/_index.md` of all the translations, so it doesn't add more work to add it in `base-l10n` too and avoid a more complicated explanation for the for PR such as "you have to copy the `base-l10n` dir and the copy the `/docs/_index.md` file from \en\". But you may disagree :-)